### PR TITLE
Fix box.stat.net() traversal for recent tarantools

### PR DESCRIPTION
**Pull request by Satbek**

### CODE REVIEW

File: stat/init.lua [hunk 1]
```diff
@@ -138,9 +138,11 @@ local function stat(args)
     do
         local i = box.stat.net()
         for k, v in pairs(i) do
-            stats['stat.net.' .. k:lower() .. '.total'] = v.total
-            stats['stat.net.' .. k:lower() .. '.rps'] = v.rps
-        end
+            if type(v) == "table" then
```
Comment (Totktonada) [inline stat/init.lua]: Nit: single quotes is used around, so it would be looks better if you'll use single quotes here too.

File: stat/init.lua [hunk 2]
```diff
@@ -262,7 +264,7 @@ local function check_replica(args)
     elseif args.include then
         for _, status in ipairs(args.include) do
             include[status] = true
-        end
+       	end
```
Comment (Totktonada) [inline stat/init.lua]: Broken indent.

File: stat/init.lua [hunk 3]
```diff
@@ -138,9 +138,11 @@ local function stat(args)
     do
         local i = box.stat.net()
         for k, v in pairs(i) do
-            stats['stat.net.' .. k:lower() .. '.total'] = v.total
-            stats['stat.net.' .. k:lower() .. '.rps'] = v.rps
-        end
+            if type(v) == "table" then
+	        stats['stat.net.' .. k:lower() .. '.total'] = v.total
```
Comment (Totktonada) [inline stat/init.lua]: Broken indent.

**Review by Totktonada — CHANGES_REQUESTED** [review]
Please, fix comments below and squash commits.

**Comment by Totktonada** [comment]
It would be good to provide more specific title for the PR, like 'Fix box.stat.net() traversal for recent tarantools' or kinda.

**Comment by rosik** [comment]
I think it would be better to explicitly traverse keys `SENT` and `RECEIVED` instead of using `pairs`.
It would be easier to understand the code.

**Comment by Totktonada** [comment]
@rosik Good idea.

File: stat/init.lua [hunk 4]
```diff
@@ -137,7 +137,7 @@ local function stat(args)
 
     do
         local i = box.stat.net()
-        for k, v in pairs(i) do
+        for k, v in pairs({SENT = i.SENT, RECEIVED = i.RECEIVED}) do
```
Comment (Totktonada) [inline stat/init.lua]: Tricky way :) As far as I understand Yaroslav proposed to do it in the most obvious way:

```lua
local box_stat_net = box.stat.net()
stats['stat.net.SENT.total'] = box_stat_net.SENT.total
stats['stat.net.SENT.rps'] = box_stat_net.SENT.rps
stats['stat.net.RECEIVED.total'] = box_stat_net.RECEIVED.total
stats['stat.net.RECEIVED.rps'] = box_stat_net.RECEIVED.rps
```

**Pull request merged**